manifests/config.pp

```diff
@@ -254,6 +254,27 @@
         notify  => Class['Rabbitmq::Service'],
       }
     }
+    'Archlinux': {
```

Comment:
this isn't the cleanest solution, but it works good. Later on we can think about unifying this setup for all distributions with systemd. Personally I prefer the camptocamp/systemd module for this, but maybe we need to wait for the discussion in https://github.com/voxpupuli/puppet-prometheus/pull/90

Reply:
Since this isn't part of the modulesync or 'spec test fixes', does it belong in a separate PR (so that is makes it into the changelog)?

Reply:
that's why I didn't label it with modulesync. It will be in the changelog.

Reply:
To stop anyone else labelling it later, I've renamed the PR to make it a bit more obvious.